docs/notebooks/03_merge.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>what would these genes be... If you know which genome annotation has been used, why would there be any gene symbol that is not resolved? Or are you talking about ensembl ids that have been removed/added between versions? </p><p> </p><p>But we can't really do anything about them, can we? </p><p> </p><p>In any case, good to investigate unmapped identifiers as a sanity check, but I wouldn't go into details of manually resolving them. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>ok, makes sense. </p>